CourseApp/Program/Zoo/Abstract/Animal.cs

```diff
@@ -0,0 +1,21 @@
+namespace CourseApp.Program.Zoo.Abstract
+{
+    public abstract class Animal
+    {
+        public Animal(string type, string size, int age, string ability)
+        {
+            Type = type;
+            Size = size;
+            Age = age;
+            Ability = ability;
+        }
+
+        public string Type { get; set; }
+
+        public string Size { get; set; }
+
+        public int Age { get; set; }
+
+        public string Ability { get; set; }
```

Comment:
let's add some common method(s) for example `eat`

and 1 abstract method - for example `makeVoice`